src/App.js

```diff
@@ -36,24 +58,50 @@ class App extends Component {
                         ]
                     }
                 ],
-                labels: ['React', 'Node', 'Angular', 'jQuery', 'SASS']
+                labels: ['Developer Love', 'Global Job Demand', 'US Job Demand', 'Startup Job Demand', 'Remote Job Demand']
             }
         });
     }
 
+    onLangClick(lang) {
+        this.setState({
+            currentLang: lang
+        }, () => {
+            this.getData();
+        });
+        console.log(this.state.currentLang)
+    }
+
+    dataExtractor() {
+        for (let i = 0; i < chartData[0].length; i++) {
+            this.state.langArray.push(chartData[0][i].name);
```

Comment:
We don't want to update state this way in React, and we should be using this.setState to update state due to the internal workings in Javascript.